python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -84,9 +84,9 @@ def test_read_json_nested(tmp_path):
         }
     )
     kwargs = dict(orient="records", lines=True)
-    with tmp_path / "data.json" as f, dask.config.set(
-        {"dataframe.convert-string": False}
-    ):
+    f = tmp_path / "data.json"
+    f.touch()
```

Comment:
*question*: Why do we need to touch the file first?

Reply:
Ah turns out we didn't need to touch the file. Removed